packages/story-editor/src/components/floatingMenu/elements/fontSize.js

```diff
@@ -17,20 +17,50 @@
 /**
  * External dependencies
  */
-import { Icons } from '@googleforcreators/design-system';
 import { __ } from '@googleforcreators/i18n';
 
 /**
  * Internal dependencies
  */
-import { IconButton } from './shared';
+import { useStory } from '../../../app';
+import {
+  focusStyle,
+  getCommonValue,
+  inputContainerStyleOverride,
+} from '../../panels/shared';
+import { MIN_MAX } from '../../panels/design/textStyle/font';
+// TODO: https://github.com/GoogleForCreators/web-stories-wp/issues/10799
+import { Input } from './shared';
 
 function FontSize() {
+  const { selectedElements, updateSelectedElements } = useStory(
+    ({ state: { selectedElements }, actions: { updateSelectedElements } }) => {
+      return {
+        selectedElements,
+        updateSelectedElements,
+      };
+    }
+  );
+
+  const fontSize = getCommonValue(selectedElements, 'fontSize');
```

Comment:
💭 if this is only shown for one element, could we pull the font size off the selected element?

Reply:
Ah yes. Nice catch! I had originally thought we would support multiple selected elements, but that is not the case.

Reply:
I had pulled this directly from our design panel font size.